### PR TITLE
Fixing lint errors and package deps

### PR DESCRIPTION
**Pull request by fmrico**
Hi all,

This PR fixes CI errors

Best
---
Signed-off-by: Francisco Martin Rico <fmrico@gmail.com>

### CODE REVIEW

File: mocap4ros_composers/mocap_camera_composer/src/mocap_camera_composer.cpp [hunk 1]
```diff
@@ -50,9 +51,11 @@ void MocapCameraComposer::marker_to_tf(const mocap4ros_msgs::msg::Markers::Share
     // RCLCPP_INFO(camera_composer_node->get_logger(), "marker name %s", marker.marker_name);
     marker_id++;
     tf2::Transform transform;
-    transform.setOrigin(tf2::Vector3(marker.translation.x / 1000,
-      marker.translation.y / 1000,
-      marker.translation.z / 1000)
+    transform.setOrigin(
+      tf2::Vector3(
```
Comment (fmrico) [inline mocap4ros_composers/mocap_camera_composer/src/mocap_camera_composer.cpp:55]: Esto me parece mal

Comment (fmrico) [reply mocap4ros_composers/mocap_camera_composer/src/mocap_camera_composer.cpp:55]: pues está bien

**Comment by lbajo** [comment]
LGTM and the CI is working! Merging

**Comment by fmrico** [comment]
We should enable tests again.

**Comment by lbajo** [comment]
I added the tests in a new PR and is ok! :)

**Pull request merged**